content/reference/glossary/cold-atom.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Cold atom"
+[extra]
+category = "hoon-nock"
++++
+
+A **cold atom** is an [atom](/reference/glossary/atom) whose type is an exact
+value equal to itself. This in contrast to a [warm
+atom](/reference/glossary/warm-atom), whose type is a general
+[aura](/reference/glossary/aura). Cold atoms have a `%` prefix, for example
+`%foo`, `%123`, `%0xdead.beef`, etc.
+
+### Further Reading
+
+- [Hoon School](/guides/core/hoon-school/): Our guide to learning the Hoon
+  programming language.
+  - [“Hoon Syntax”](/guides/core/hoon-school/B-syntax#nouns): A Hoon School
+    lesson that explains how atoms work.
```

Comment:
and include:

- [Atoms and strings](https://developers.urbit.org/reference/hoon/rune/constants)